internal/provider/data_source_cloudexport_item.go

```diff
@@ -21,7 +21,7 @@ func dataSourceCloudExportItemRead(ctx context.Context, d *schema.ResourceData,
 		ExportGet(ctx, d.Get("id").(string)).
 		Execute()
 	if err != nil {
-		return detailedDiagError("Failed to read cloud export item", err, httpResp)
+		return detailedDiagError("failed to read cloud export item", err, httpResp)
```

Comment:
_Minor_: Is the initial lowercase letter dictated by some linter? If not, why do you want to change it?

Reply:
I did it to make it the same as in synthetics provider. Errors in Go are not capitalized, because they're often concatenated into one error message.
But in that case this is not error string, but a summary which is not used that way, so I'll change the form to capitalized in both providers.

Reply:
Done